locales/en/messages.json

```diff
@@ -3383,12 +3383,39 @@
     "powerBatteryWarning": {
         "message": "Warning Cell Voltage"
     },
+    "powerCalibrationManager": {
+        "message": "Calibration Manager"
```

Comment:
'Calibration Manager' is misleading here, as this is not really a 'manager'. I'd just use 'Calibrate'.